repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -1,67 +1,106 @@
 
 const title = 'Exports';
 const description = 'View and access your settings in one place.';
 
 interface GlobalSettingsProps {
-    exports: { matchingDataCount: number; name: string; netexCount: number; signedUrl: string | null }[];
+    exports: {
+        matchingDataCount: number;
+        name: string;
+        netexCount: number;
+        signedUrl: string | null;
+        exportDate: string;
+        exportTime: string;
+    }[];
     csrf: string;
+    myFaresEnabled: boolean;
+    exportEnabled: boolean;
+    operatorHasProducts: boolean;
 }
 
-const Exports = ({ exports, csrf }: GlobalSettingsProps): ReactElement => {
+const Exports = ({
+    exports,
+    csrf,
+    myFaresEnabled,
+    exportEnabled,
+    operatorHasProducts,
+}: GlobalSettingsProps): ReactElement => {
     return (
         <>
-            <BaseLayout title={title} description={description} showNavigation>
+            <BaseLayout
+                title={title}
+                description={description}
+                showNavigation
+                myFaresEnabled={myFaresEnabled}
+                exportEnabled={exportEnabled}
+            >
                 <div className="govuk-grid-row">
-                    <div className="govuk-grid-column-three-quarters">
-                        <h1 className="govuk-heading-xl">Exports</h1>
-
-                        <CsrfForm csrfToken={csrf} method={'post'} action={'/api/exports'}>
-                            <button type="submit" className="govuk-button">
-                                Export all
-                            </button>
-                        </CsrfForm>
+                    <div className="govuk-grid-column-full">
+                        <div className="dft-flex dft-flex-justify-space-between">
+                            <h1 className="govuk-heading-m">Export your data</h1>{' '}
+                            <CsrfForm csrfToken={csrf} method={'post'} action={'/api/exports'}>
+                                {operatorHasProducts && (
+                                    <button type="submit" className="govuk-button">
+                                        Export all fares
+                                    </button>
+                                )}
+                            </CsrfForm>
+                        </div>
 
+                        <p className="govuk-body-m dft-four-fifths-width">
+                            This will export all of your current active products and any pending products. Any products
+                            flagged with ‘Needs Attention’ will be exported but may not be correct. Expired products or
+                            products for expired services, will not be included in the export. Once your export is
+                            complete, it will appear in the list below and you can download this and upload it to{' '}
+                            <a href="https://publish.bus-data.dft.gov.uk/">BODS</a>.
+                        </p>
```

Comment:
Some margin/spacing between this and the table.

Reply:
Can you help with this please?